### DISCUSSION TITLE
要是能直接在APP上发帖子就好了

### DISCUSSION THREAD

**Discussion by LeoneMafia** [share121/inter-knot#5399] [其他]
也不知道是我操作不对还是怎么回事，我都不知道怎么在APP上登录，注册也是在网站上弄的，换头像也不知咋整，家人们有人教教吗？也可能是我光顾着倒腾这个忘了看别的？

**Comment by Ritanlisa** [comment]
应该都读取的github吧

**Reply by LeoneMafia** [reply]
这边刚弄好了，就是不知怎么上传自己的头像，也不知为啥，貌似发帖子还得在网页版发

**Comment by LeoneMafia** [comment]
好吧这边没事了，刚看到一位大佬的帖子把这些都解释清了，抱歉麻烦了
